Problem1.rb

```diff
@@ -0,0 +1,18 @@
+def  sum_even_numbers(array)
+
+  sum = 0
+  array.each { |x| sum += x.even? ? x : 0 }

```

Comment:
Can you make it one line (without any aux variable)?

Reply:
Yes: array.select(&:even?).sum
Commit: d2d8a7489a6358e37802db6ee26054408c72efad